dolphin/api/v1/storages.py

```diff
@@ -192,12 +192,13 @@ def _set_synced_if_ok(context, storage_id, resource_count):
               % storage_id
         raise exception.InvalidInput(message=msg)
     else:
-        last_update = storage['updated_at']
-        current_time = datetime.now()
+        last_update = storage['updated_at'] or storage['created_at']
```

Comment:
Why do we consider UTC time? Will it make any difference for different timezones for consumers?
Will considering time as Epoch time everywhere will make sense?

Reply:
The reason to use UTC time is that the oslo_db use UTC time to set time to 'update_at' and 'created_at'.
So no matter what is the timezone, if we use the same interface to get the current time as what oslo_db use, we can get the correct interval, otherwise we may get the incorrect interval because of the jet lag.